base_images/isce2/docker/Dockerfile

```diff
@@ -31,6 +31,7 @@ WORKDIR /projects
 RUN sed -i -e 's/\/root/\/projects/g' /etc/passwd
 
 RUN conda install -y -c conda-forge mamba=1.3.1
+RUN pip install pyOpenSSL==23.2.0
```

Comment:
any reason to use pip here and conda in the other image?

Reply:
Both should be pip was changed back